Kitodo/src/test/resources/selenium/resources/hibernate.cfg.xml

```diff
@@ -22,11 +22,14 @@
         <!-- SQL - Settings -->
         <property name="dialect">org.hibernate.dialect.H2Dialect</property>
         <property name="hibernate.connection.driver_class">org.h2.Driver</property>
+
         <property name="hibernate.connection.username">sa</property>
         <property name="hibernate.connection.password"/>
-        <property name="hibernate.connection.url">
-            jdbc:h2:tcp://localhost/mem:test;DB_CLOSE_DELAY=-1
-        </property>
+        <property name="hibernate.connection.url">jdbc:h2:tcp://localhost/mem:test;DB_CLOSE_DELAY=-1;MODE=MariaDB;DATABASE_TO_LOWER=TRUE;CASE_INSENSITIVE_IDENTIFIERS=TRUE;NON_KEYWORDS=USER</property>
```

Comment:
Just out of curiosity: is this additional `tcp` in the connection url required because Selenium accesses the database via the browser e.g. http, thus tcp?

Reply:
I did not see an additional connection uri. It is the same as before only with the needed additional parameters like on the other places. Why only a tcp based connection is working on the selenium configuration is out of my knowledge but was before this changes already in use. Maybe the tcp connection is used here because of the used Tomcat container where no other connections are possible.
Or did I read your comment wrong?

Reply:
Ah, yes, you are right, the `tcp` was already there, I missed that.